packages/circuit-ui/components/NotificationFullscreen/__snapshots__/NotificationFullscreen.spec.tsx.snap

```diff
@@ -1,5 +1,437 @@
+  justify-content: center;
+  width: auto;
+  height: auto;
+  margin: 0;
+  cursor: pointer;
+  text-align: center;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  font-weight: 700;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 999999px;
+  -webkit-transition: opacity 120ms ease-in-out,color 120ms ease-in-out,background-color 120ms ease-in-out,border-color 120ms ease-in-out;
+  transition: opacity 120ms ease-in-out,color 120ms ease-in-out,background-color 120ms ease-in-out,border-color 120ms ease-in-out;
+  background-color: #3063E9;
+  border-color: #3063E9;
+  color: #FFF;
+  padding: calc(12px - 1px) calc(24px - 1px);
+  position: relative;
+  overflow: hidden;
+}
+
+.circuit-10:focus {
+  outline: 0;
+  box-shadow: 0 0 0 4px #AFD0FE;
+}
+
+.circuit-10:focus::-moz-focus-inner {
+  border: 0;
+}
+
+.circuit-10:focus:not(:focus-visible) {
+  box-shadow: none;
+}
+
+.circuit-10:disabled,
+.circuit-10[disabled] {
+  opacity: 0.5;
+  pointer-events: none;
+  box-shadow: none;
+}
+
+.circuit-10:hover {
+  background-color: #234BC3;
+  border-color: #234BC3;
+}
+
+.circuit-10:active,
+.circuit-10[aria-expanded='true'],
+.circuit-10[aria-pressed='true'] {
+  background-color: #1A368E;
+  border-color: #1A368E;
+}
+
+.circuit-14 {
+  font-size: 16px;
+  line-height: 24px;
+  display: -webkit-inline-box;
+  display: -webkit-inline-flex;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  -webkit-justify-content: center;
+  justify-content: center;
+  width: auto;
+  height: auto;
+  margin: 0;
+  cursor: pointer;
+  text-align: center;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  font-weight: 700;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 999999px;
+  -webkit-transition: opacity 120ms ease-in-out,color 120ms ease-in-out,background-color 120ms ease-in-out,border-color 120ms ease-in-out;
+  transition: opacity 120ms ease-in-out,color 120ms ease-in-out,background-color 120ms ease-in-out,border-color 120ms ease-in-out;
+  padding: calc(12px - 1px) calc(24px - 1px);
+  background-color: transparent;
+  border-color: transparent;
+  color: #3063E9;
+  padding-left: 0;
+  padding-right: 0;
+  position: relative;
+  overflow: hidden;
+  margin-top: 16px;
+}
+
+.circuit-14:focus {
+  outline: 0;
+  box-shadow: 0 0 0 4px #AFD0FE;
+}
+
+.circuit-14:focus::-moz-focus-inner {
+  border: 0;
+}
+
+.circuit-14:focus:not(:focus-visible) {
+  box-shadow: none;
+}
+
+.circuit-14:disabled,
+.circuit-14[disabled] {
+  opacity: 0.5;
+  pointer-events: none;
+  box-shadow: none;
+}
+
+.circuit-14:hover {
+  color: #234BC3;
+}
+
+.circuit-14:active,
+.circuit-14[aria-expanded='true'],
+.circuit-14[aria-pressed='true'] {
+  color: #1A368E;
+}
+
+@media (min-width: 480px) {
+  .circuit-14 {
+    display: none;
+  }
+}
+
+<div>
+  <div
+    class="circuit-0"
+  >
+    <div
+      class="circuit-1"
+    >
+      <svg
```

Comment:
I think this would get announced "image"

Reply:
Should be solved with aria-hidden that is passed to an svg element if the alt value is `""`. Will verify with manual testing

Reply:
Chrome x VoiceOver:

| | img | svg |
| --- | --- | --- |
| alt | <video src="https://user-images.githubusercontent.com/35560568/189392461-e07195ec-fe80-4426-904d-c113011207de.mov" /> | <video src="https://user-images.githubusercontent.com/35560568/189392497-63af9056-a4db-4160-9adb-540e43788e35.mov" /> |
| !alt | <video src="https://user-images.githubusercontent.com/35560568/189392485-c49fada1-b429-4e11-91f3-820ad71a2e85.mov" /> | <video src="https://user-images.githubusercontent.com/35560568/189392514-51970427-ae01-4efd-8865-cbd67810ba40.mov" /> |

TL;DR: everything looks good with this user agent. I think we can roll with it